src/main/webapp/scripts/map.js

```diff
@@ -26,14 +55,40 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  /*Add Hard-Coded Markers */
+  addMarker(map);
+}
+
+/* Temporary Function to Hold Marker for Sky Garden */
+function addMarker(map) {
+  var latLng = {lat: 51.510881, lng: -0.083751}
+  var marker = new google.maps.Marker({
+      position: latLng,
+      map: map,
+      title: "Sky Garden"
+      /*icon: blueIcon*/
+  })
+  marker.addListener("click", () => {
+        document.getElementById("rec-container").style.display = "block";
+        map.setZoom(16);
+        map.setCenter(marker.getPosition());
+  });
 }
 
 /* Place Marker Where Map is Clicked On & Show Popup*/
 function placeMarkerAndPanTo(latLng, map) {
+  const greyIcon = {
+    url: "/images/grey-marker.png", // url
+    scaledSize: new google.maps.Size(30, 40), // scaled size
+    origin: new google.maps.Point(0, 0), // origin
+    anchor: new google.maps.Point(0, 0) // anchor 
+  }
```

Comment:
please move this part to the sections with the other  _Icon

Reply:
It can't be moved as there is an error currently with the other icons. For now, it's fine where it is, as this is only needed within the function to place an icon.

Reply:
it's more convenient to have it alongside other constants above, I'm curious to know what was the error of having this as a global constant?